src/sentry/static/sentry/app/views/settings/projectDataPrivacy/projectDataPrivacyContent.tsx

```diff
@@ -74,9 +76,16 @@ class ProjectDataPrivacyContent extends AsyncView<Props> {
             ]}
           />
         </Form>
-        <ProjectDataPrivacyRulesPanel
-          orgId={orgId}
-          projectId={projectId}
+        <DataPrivacyRulesPanel
+          panelHeaderSubTitle={
+            <PanelHeaderSubTitle>
+              {t('this can also be configured org-wide in')}
+              <Link to={`/settings/${orgId}/security-and-privacy/`}>
+                {t('Organization Security and Privacy')}
+              </Link>
```

Comment:
Actually even better would be if we used `tct`, this gives translator more context when translating


e.g.

```suggestion
              {tct('This can also be configured org-wide in [linkToOrganizationSecurity]', {
              	linkToOrganizationSecurity: <Link to={`/settings/${orgId}/security-and-privacy/`}>
                {t('Organization Security and Privacy')}
              </Link>
              })}
```

This way translators will see:

`this can also be configured org-wide in [linkToOrganizationSecurity]`